button-merchant/src/main/java/com/usebutton/merchant/ButtonUtil.java

```diff
@@ -73,4 +85,8 @@ public static String sha256Encode(String value) {
 
         return null;
     }
+
+    public static boolean isValidEmail(String email) {
+        return Pattern.compile(EMAIL_REGEX).matcher(email).matches();
```

Comment:
I would put the compilation inside a `static` block so that you don't do it each time you need to verify.